cluster-defns/hopsworks-head.yml

```
@@ -43,6 +43,9 @@ attrs:
   prometheus:
       retention_time: "8h"
   hopsworks:
+    enable_flyingduck: true
+    enable_jupyter_python_kernel_non_kubernetes: true
+    python_kernel: true
     hopsworks_public_host: __HOPSWORKS_PUBLIC_IP__  
```

Maybe you want to add the localhost loadbalancer by default as well

```suggestion
    hopsworks_public_host: __HOPSWORKS_PUBLIC_IP__  
    loadbalancer_external_domain: localhost
```

---

It's the __HOPSWORKS_PUBLIC_IP__ i need to add here. 
Will do it.